cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -46,18 +46,19 @@ type CloudControllerManagerServer struct {
 func NewCloudControllerManagerServer() *CloudControllerManagerServer {
 	s := CloudControllerManagerServer{
 		KubeControllerManagerConfiguration: componentconfig.KubeControllerManagerConfiguration{
-			Port:                    ports.CloudControllerManagerPort,
-			Address:                 "0.0.0.0",
-			ConcurrentServiceSyncs:  1,
-			MinResyncPeriod:         metav1.Duration{Duration: 12 * time.Hour},
-			NodeMonitorPeriod:       metav1.Duration{Duration: 5 * time.Second},
-			ClusterName:             "kubernetes",
-			ConfigureCloudRoutes:    true,
-			ContentType:             "application/vnd.kubernetes.protobuf",
-			KubeAPIQPS:              20.0,
-			KubeAPIBurst:            30,
-			LeaderElection:          leaderelectionconfig.DefaultLeaderElectionConfiguration(),
-			ControllerStartInterval: metav1.Duration{Duration: 0 * time.Second},
+			Port:                      ports.CloudControllerManagerPort,
+			Address:                   "0.0.0.0",
+			ConcurrentServiceSyncs:    1,
+			MinResyncPeriod:           metav1.Duration{Duration: 12 * time.Hour},
+			NodeMonitorPeriod:         metav1.Duration{Duration: 5 * time.Second},
+			ClusterName:               "kubernetes",
+			ConfigureCloudRoutes:      true,
+			ContentType:               "application/vnd.kubernetes.protobuf",
+			KubeAPIQPS:                20.0,
+			KubeAPIBurst:              30,
+			LeaderElection:            leaderelectionconfig.DefaultLeaderElectionConfiguration(),
+			ControllerStartInterval:   metav1.Duration{Duration: 0 * time.Second},
+			RouteReconciliationPeriod: metav1.Duration{Duration: 10 * time.Second},
```

Comment:
Please add a comment to this function that many values are copied from the original controller manager and should probably stay in syc, and add a reciprocal comment to the original location.

Reply:
Sounds good. Note added.